### PR TITLE
Wagtail 4.0 support

### PR DESCRIPTION
**Pull request by katdom13**
- Updated to support wagtail 4
- Dropped support for versions of Wagtail less than 2.15
- Updated documentation
- Added custom template for checking wagtail routes (e.g. wagtailsnippets)
- Fixed imports
- Fixed formatting
- Added tox.ini
- Updated ci testing with tox to properly manage versions

```
Found 56 test(s).
Creating test database for alias 'default'...
System check identified some issues:

WARNINGS:
tests.Advert.publications: (fields.W340) null has no effect on ManyToManyField.

System check identified 1 issue (0 silenced).
.......................................................4 object created. 0 object updated. 0 object skipped.
4 items created. 0 items updated. 0 items skipped.
.
----------------------------------------------------------------------
Ran 56 tests in 0.334s

OK
```

### CODE REVIEW

**Review by nickmoreton — APPROVED** [review]
This looks like a good set of changes. Nice work

**Comment by jacobtoppm** [comment]
Hey, apologies this was mixed @katdom13! Would you be able to fix conflicts and then we can get this merged?

**Comment by katdom13** [comment]
Hi @jacobtoppm ,
I have fixed the conflicts. Can you please approve the workflow approval for this?
Thank you!

File: wagtail_airtable/templatetags/wagtail_airtable_tags.py [hunk 1]
```diff
@@ -18,3 +20,23 @@ def can_import_model(model_label) -> bool:
...
+
+
+@register.simple_tag
+def wagtail_route(name, label, model, *args) -> str:
+    """
+    Return the wagtail URL using the label and model.
+
+    Use:
+    {% load wagtail_airtable_tags %}
+    {% can_import_model "yourapp.ModelName" as template_var %}
```
[the remark this answers is not in the document]
Comment (katdom13) [reply wagtail_airtable/templatetags/wagtail_airtable_tags.py]: Hi @jacobtoppm ,
I have updated the doc string as follows:
https://github.com/wagtail/wagtail-airtable/pull/41/files#diff-b565bca4f4491330dfbb8d5111149a61032c3263827c52f1bc2f7314061fb2d9R27-R39

Thank you!

**Review by jacobtoppm — APPROVED** [review]

**Pull request merged**